yarn.lock

```diff
@@ -2163,10 +2175,6 @@ camelcase@^1.0.2:
   version "1.2.1"
   resolved "https://registry.yarnpkg.com/camelcase/-/camelcase-1.2.1.tgz#9bb5304d2e0b56698b2c758b08a3eaa9daa58a39"
 
-camelcase@^3.0.0:
```

Comment:
something wrong with the file dependencie

could you please remove `yarn.lock` and restart `yarn`?